vmdb/app/models/configured_system_foreman/placement.rb

```diff
@@ -0,0 +1,10 @@
+module ConfiguredSystemForeman::Placement
+  def available_configuration_profiles
+    cl_history = configuration_location.try(:path) || []
+    co_history = configuration_organization.try(:path) || []
```

Comment:
pretty sure we always have a `configuration_location`. Are you seeing otherwise?
can we remove the `try` and `|| []`?